main.py

```diff
@@ -190,10 +190,10 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(r'%s' % feconf.ADMIN_URL, admin.AdminPage),
     get_redirect_route(r'/adminhandler', admin.AdminHandler),
     get_redirect_route(r'/adminrolehandler', admin.AdminRoleHandler),
-    get_redirect_route(r'/adminjoboutput', admin.AdminJobOutput),
+    get_redirect_route(r'/adminjoboutput', admin.AdminJobOutputHandler),
     get_redirect_route(
         r'/admintopicscsvdownloadhandler',
-        admin.AdminTopicsCsvDownloadHandler),
+        admin.AdminTopicsCsvDownloadFileDownloader),
```

Comment:
Isn't there at least one more downloader somewhere in /create? Like in the exploration editor you can download old versions of explorations...